pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.apache.hugegraph</groupId>
     <artifactId>hugegraph-common</artifactId>
-    <version>1.8.10</version>
+    <version>2.0.0</version>
```

Comment:
also need to update CommonVersion.java

Reply:
the version maybe should unify together (ensure then modify them)